docs/MO_DG/prepare_model/Model_Optimizer_FAQ.md

```diff
@@ -31,15 +31,15 @@ For example, to add the description of the `CustomReshape` layer, which is an ar
 cd <SITE_PACKAGES_WITH_INSTALLED_OPENVINO>/openvino/tools/mo/front/caffe/proto
 python3 generate_caffe_pb2.py --input_proto <PATH_TO_CUSTOM_CAFFE>/src/caffe/proto/caffe.proto
 ```
-where `PATH_TO_CUSTOM_CAFFE` is the path to the root directory of custom Caffe\*.
+where `PATH_TO_CUSTOM_CAFFE` is the path to the root directory of custom Caffe.
 
-3.  Now, the Model Optimizer is able to load the model into memory and start working with your extensions if there are any.
+3.  Now, Model Optimizer is able to load the model into memory and start working with your extensions if there are any.
 
-However, because your model has custom layers, you must register your custom layers as custom. To learn more about it, refer to the section [Custom Layers in Model Optimizer](customize_model_optimizer/Customize_Model_Optimizer.md).
+However, since your model has custom layers, you must register them. To learn more about it, refer to the [Custom Layers in Model Optimizer](customize_model_optimizer/Customize_Model_Optimizer.md) section.
```

Comment:
```suggestion
However, since your model has custom layers, you must register them as custom. To learn more about it, refer to the [Custom Layers in Model Optimizer](customize_model_optimizer/Customize_Model_Optimizer.md) section.
```

Reply:
This link was https://docs.openvino.ai/2020.4/openvino_docs_MO_DG_prepare_model_customize_model_optimizer_Customize_Model_Optimizer.html in 2020.4, while this page is not there anymore since 2021. Need to find a more proper page or maybe just remove this link or paragraph.